app/models/agents/webhook_agent.rb

```diff
@@ -88,7 +88,7 @@ def receive_web_request(params, method, format)
         create_event(payload: payload)
       end
 
-      [response_message, code]
+      [interpolated(params)['response'] || interpolated['response'] || 'Event Created', code]
```

Comment:
@cantino done. But how @dsander indicated, we have no empty response option. Look at Travis ...